rebar.config

```diff
@@ -2,6 +2,7 @@
 
 {deps, [ {getopt, "1.0.1"}
        , {typerefl, {git, "https://github.com/k32/typerefl", {tag, "0.4.0"}}}
+       , {cuttlefish, {git, "https://github.com/emqx/cuttlefish.git", {tag, "v3.3.4"}}}
```

Comment:
why isn't this in test profile ?

Reply:
Schema generator `emqx_schema_cuttlefish` need it. But maybe we don't need to commit `emqx_schema_cuttlefish`, since we will not use it in the future.

Reply:
maybe add a new profile for it to create a escript tool so we can use it to convert .schema files.
ideally the entire profile can be deleted after all the conversion is done ?

Reply:
applied